examples/openrouter_example.py

```diff
@@ -0,0 +1,319 @@
+"""Example script demonstrating OpenRouter integration with GenAI Processors.
+
+This example shows how to use the OpenRouter processor to access hundreds of
+AI models through a unified API. OpenRouter provides access to models from
+OpenAI, Anthropic, Google, Meta, and many other providers.
+
+## Setup
+
+To run this example, you'll need:
+
+1. An OpenRouter API key (get one at https://openrouter.ai/keys)
+2. Install the required dependencies
```

Comment:
Drop this line: there are no extra dependencies required. The core package actually already depends on httpx, I'll add it explicitly.

If there were dependencies to be installed, we should have said here what are they or how to figure out what to install.